docs/configuration.md

```diff
@@ -3184,6 +3184,13 @@ project {
 }
 ```
 
+### `project.includeMarkdown`
```

Comment:
i don't think you need this. users can explicitly add a glob for `.md` files, see https://scalameta.org/scalafmt/docs/configuration.html#appending-to-preset-collections

Reply:
Ah-ha! I just knew something like this must be possible.
Thanks, fixed this and removed the new/unnecessary config.